cmake/find/Gettext.cmake

```diff
@@ -283,135 +281,41 @@ if(XGETTEXT_FOUND)
         endif()
       endforeach()
 
-      if(ARGS_DESKTOPFILES OR ARGS_GSETTINGSFILES)
-        find_package(Intltool REQUIRED)
+      if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
```

Comment:
I think FreeBSD and friends also use desktop files. So maybe replace this here (and elsewhere) by
`if(UNIX AND NOT APPLE)`

Reply:
I have never used (Free)BSD in a desktop environment, but in view of https://man.freebsd.org/cgi/man.cgi?query=xdg-desktop-menu&sektion=1&manpath=freebsd-release-ports
and since desktop files refer to icons and mim-types as well, I would expect that FreeBSD uses all of these files.